### ISSUE TITLE
Translations: Country menu in Index not translated

### ISSUE THREAD

**Issue by UFU608** [theCrag/website#2360]
<img width="449" alt="screen shot 2016-09-02 at 16 41 09" src="https://cloud.githubusercontent.com/assets/19162952/18196409/2b45f2da-712c-11e6-9de9-4f2b600199c0.png">

**Comment by scd** [comment]
This is user data and we are a fair way off getting this done. It was actually what I would have worked on before the templating translations if you did not come on board. See issue #1306  (II have delayed this implementation, partly because I need a break from translations and partly because there are higher priority issues)

Just for clarification this is an index area name, which just happens to match the country.

**Comment by brendanheywood** [comment]
We could treat countries as a special case because they are used outside the context of an index node, eg attached to grade systems, and used in your profile settings and other places. Something relatively simple we could do is in the mason templates if the node is if it has 'countryCode' set to something then ignore it's current name and just call ```getText($node->{countryCode})```. If we went down this path we should also do it for any regions above the country level too. Maybe we could use the node id as something unique eg ```getText('node' . $node->{id})``` but this could make it a little hard to find the right key to translate.

**Comment by scd** [comment]
Yes we can do this, but when we solve #1306 in case there are other considerations.
